src/cloudstore.js

```diff
@@ -22,9 +23,12 @@ export async function putKey({ keyId, ciphertext }) {
   if (!isId(keyId) || !isBuffer(ciphertext)) {
     throw new Error('Invalid args');
   }
-  if (await Store.getItem(KEY_ID)) {
+
+  const item = await Store.getItem(KEY_ID);
+  if (item) {
```

Comment:
What prompted the change here?

Reply:
Think I just forgot to change back after testing, will revert back 🚧